utils/ixdtf/src/parsers/tests.rs

```diff
@@ -586,6 +586,31 @@ fn valid_unambiguous_time() {
     }
 }
 
+#[test]
+fn ambiguous_annotations() {
+    const TESTS_TIMEZONE: &[&str] = &[
+        "2020-01-01[Asia/Kolkata]",
+        // Has a slash
```

Comment:
they both have a slash though

Reply:
oops, first should say "starts with capital letter"